### PR TITLE
#378 Add instruction to update Valids table

### PR DESCRIPTION
**Pull request by anton-lisanin**

### CODE REVIEW

File: evm_loader/program/src/instruction.rs [hunk 1]
```diff
@@ -353,6 +359,7 @@ impl<'a> EvmInstruction<'a> {
                 let step_count = step_count.try_into().ok().map(u64::from_le_bytes).ok_or(InvalidInstructionData)?;
                 EvmInstruction::ExecuteTrxFromAccountDataIterativeV02 {collateral_pool_index, step_count}
             },
+            23 => EvmInstruction::UpdateValidsTable,
```
Comment (vasiliy-zaznobin) [inline evm_loader/program/src/instruction.rs:362]: Does the indexer know about the new instruction?

File: evm_loader/program/src/entrypoint.rs [hunk 2]
```diff
@@ -744,6 +744,34 @@ fn process_instruction<'a>(
 
             do_write(holder_info, offset, bytes)
         },
+        EvmInstruction::UpdateValidsTable => {
```
Comment (s-medvedev) [inline evm_loader/program/src/entrypoint.rs:747]: We need a possibility to send this transaction to evm_loader. Maybe add a command `update-valids-table` in `neon-cli`?

File: evm_loader/program/src/instruction.rs (hunk 1 above)
Comment (s-medvedev) [reply evm_loader/program/src/instruction.rs:362]: This instruction doesn't influence to execution model in the indexer.
But to be totally correct indexer should process it.
I created the issue neonlabsorg/proxy-model.py#301

**Review by s-medvedev — DISMISSED** [review]

File: evm_loader/program/src/entrypoint.rs (hunk 2 above)
Comment (anton-lisanin) [reply evm_loader/program/src/entrypoint.rs:747]: Done

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**